color.html

```diff
@@ -75,12 +98,43 @@
       --lumo-success-text-color: hsl(145, 100%, 32%);
       --lumo-success-contrast-color: #FFF;
     }
+    /* MAGI REMOVE START */
   </style>
 </custom-style>
+<!-- MAGI REMOVE END -->
+<!-- MAGI ADD START
+`;
+const $tpl = document.createElement('template');
+$tpl.innerHTML = `<custom-style><style>${colorBase.toString().replace(':host', 'html')}</style></custom-style>`;
+document.head.appendChild($tpl.content);
+
+Vaadin.LumoStyles.colorBase = colorBase;
+})();
+</script>
+
+MAGI ADD END -->
+
+<!-- MAGI ADD START
```

Comment:
Looks like an unnecessary END and START here. Not a blocker.

Reply:
Removed.